django/thunderstore/community/api/experimental/urls.py

```diff
@@ -17,6 +18,11 @@
         PackageCategoriesExperimentalApiView.as_view(),
         name="categories",
     ),
+    path(
+        "community/<slug:community_id>/validate-packages/",
```

Comment:
I'm somewhat conflicted with making this community scoped. Perhaps it's the easiest way to do it, as the alternative would be to make it global and then introduce a parameter that can optionally be used to filter by community, but I feel that's unnecessary complexity for now. This is the experimental API after all.

So while this isn't really aligned with long term goals of keeping packages community-agnostic, it's probably the solution we want to move forward with and no changes are needed 👍

Reply:
I think the validation (for this use case) should be community specific, since AFAIK the mod manager can't install packages that aren't listed under the currently selected game/community.